### DISCUSSION TITLE
Issue publish basic WASM app to Github Pages

### DISCUSSION THREAD

**Discussion by dbriard** [AvaloniaUI/Avalonia#16694] [Q&A] [answered]
Hi, I am trying to publish an app created with Avalonia 11.1.3 to Github Pages and I run into 404 issues with all files inside the _framework folder.

![image](https://github.com/user-attachments/assets/c2f22f74-9c2f-4903-9964-291a33c3638f)

Here what I did:
1. I created a new repository on Github
2. I called "dotnet publish" in the browser folder of my app
3. I tested using dotnet serve that the published site work well, and it is.
4. I copied all the files from .....Browser\bin\Release\net8.0-browser\publish\wwwroot to the root if the Github repository.
![image](https://github.com/user-attachments/assets/948a836d-a9de-4930-9984-aca0b25fb7f0)
5. Pushed on Github, then settings > pages > and builded the page from the main branch.

I previously managed to publish app in Github page but it was before .NET8 and Avalonia 11.1. By the time, I remember I had to change the urls inside index.html from ./file to ./githubproject/file
But now the structure of the project changed, and there is the _framework folder.

Anyone has any idea of why the files in _framework throw a 404 error, they are present in the repository.
Did I forgot an important step?

Thanks in advance!

**Comment by rabbitism** [comment, marked as answer]
IIRC you need to add an empty .nojekyll file like this
 https://github.com/irihitech/Semi.Avalonia/blob/gh-pages/.nojekyll

Otherwise all resources start with an underscore will be ignored. (github pages behavior)

**Reply by dbriard** [reply]
It works like a charm :)
A big thanks to you for your help!
